guides/common/modules/proc_implementing-pxe-less-discovery.adoc

```diff
@@ -28,11 +28,9 @@ ifdef::orcharhino[]
 image::common/pxeless-mode-orcharhino.svg[PXE-less mode]
 endif::[]
 
-ifndef::satellite,orcharhino[]
-image::common/pxeless-mode.svg[]
-endif::[]
```

Comment:
Was it intentional to remove the image for Foreman/Katello builds? IMHO these lines should not affect Satellite.

Reply:
That must a be a mistake, thanks!